docs/getting-started/glossary.md

```diff
@@ -1,69 +1,77 @@
 # Glossary & Key Concepts
 
-This page ldefines essential terms used throughout the Roar User Guide.  
+This page idefines essential terms used throughout the Roar User Guide.  
 Understanding these concepts will help you use the system efficiently and communicate clearly with support.
 
+## Computing Cluster
+A computing cluster is a collection of interconnected computers (nodes) that collaborate to perform computational tasks. The Roar computing cluster is a high-performance computing (HPC) system designed for research, comprising numerous nodes equipped with CPUs, GPUs, memory, and storage, all coordinated by the Slurm workload manager to process jobs efficiently.
 
 ## Node
-
-A **node** is a single physical computer in the Roar cluster.  
+A Node is a single physical computer in the Roar cluster.  
 Each node has its own:
+
 - CPU(s)
 - Memory (RAM)
 - Local storage (usually temporary)
 - Network interface
 
-Think of a node as a **complete server** dedicated to running jobs.  
-Multiple jobs can run on one node if resources allow.
-
-**Example**:  
-> “Your job is running on node `gpu-a100-03`.”
+Think of a node as a complete server dedicated to running jobs.  Multiple jobs can run on one node if resources allow.
+Example > “Your job is running on node `submit03`”
 
+## Processor
+A processor (or CPU, Central Processing Unit) is the hardware component within a node responsible for executing computational tasks. A single processor contains multiple cores, each capable of running independent threads. For example, a node with two processors, each with 32 cores, provides 64 cores total.
 
 ## Core
-
-A **core** is a single processing unit within a CPU.  
-Modern CPUs have multiple cores (e.g., 64 cores per node).
-
+A Core is a single processing unit within a CPU. Modern CPUs have multiple cores (e.g., 64 cores per node).
 Cores execute tasks in parallel.  
 When you request “8 cores,” you’re asking for 8 processing units — possibly on one node or across nodes.
+Note:  
 
-**Note**:  
 - 1 core = 1 CPU thread (unless hyper-threading is enabled)
-- More cores = faster parallel computation (if your code supports it)
-
-
-## Directory
-
-A **directory** is a folder in the filesystem — a location where files are stored.
-
-On Roar, key directories include the home, work, group and scratch directories.
-
-See [File Storage & Filesystems](../file-system/file-storage.md) for full details.
-
+- Parallel performance depends on your application’s ability to utilize multiple cores.
```

Comment:
We should either add a definition for "Parallel processing" (and conversely serial processing), or we should find a way to reword this with less jargon.